### PR TITLE
added `fallback` value since find-namespace-value supports this

### PR DESCRIPTION
**Pull request by hellatan**

### CODE REVIEW

**Comment by hellatan** [comment]
can't assign to anyone cc @twhid @antialias

**Comment by antialias** [comment]
I don't have a super-strong opinion about this, but it seems to me that `SV.get(foo) || bar` works well enough as a pattern for ensuring a fallback to get `bar` if `foo` does not exist in serverVars.

Also, if we are going to support a fallback feature, I'd prefer that it be set as an option:
```js
SV.get(foo, {fallback: bar})
```

**Comment by hellatan** [comment]
that's not the signature to the find namespace value. Though, i'm converting that module to es6 instead and changing the signature completely to be an object instead of a list of arguments anyway. this only matters if we want to get this functionality in before that happens.

**Comment by twhid** [comment]
I agree with TH, doesn't seem like something that needs to be in the
module. So many other ways to get a default value if that's what you want.
Seems like a job for the consumer, doesn't need to be part of the api.

**Comment by hellatan** [comment]
ok. since i'm refactoring the FNV module, what about what @antialias said and adding it in as an option? Should i keep that functionality at all then?

**Comment by antialias** [comment]
I'm not eager for this functionality and don't think I'll use it much if ever, but if you really really want to add it, go for it.

**Comment by antialias** [comment]
(edited my comment on github to reflect a slightly different sentiment)

**Comment by twhid** [comment]
I agree w @antialias -- it you find it useful go for it. Adding it as an option seems to be the least invasive way to add it.

File: test/Client_spec.js [hunk 1]
```diff
@@ -34,4 +34,8 @@ describe('serverVars client side', function () {
         assert.equal(serverVars.get('test1'), 1);
         assert.equal(serverVars.get('test2.test3'), 3);
     });
+    it('should use the `fallback` value', function () {
+        var value = serverVars.get('noValue', 100);
+        assert.equal(value, 100);
+    })
```
Comment (antialias) [inline test/Client_spec.js]: missing semicolon at EOL

Comment (hellatan) [inline test/Client_spec.js]: damn it. need to setup a linter here. I'm going to close this PR, actually.

**Pull request closed without merging**